frontend/src/components/Number.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
```

Comment:
It that `Number` component used somewhere ?

Reply:
No. I'll delete them before merging